src/tools/util/consoleargs.cpp

```diff
@@ -103,12 +104,12 @@ DWORD GetCanonFilePath(_In_z_ LPCWSTR wszSrcFileName, _Out_z_cap_(cchDestFileNam
     // devices beginning with "\\.\"
     // or wildcards
     // or characters 0-31
-    if (wcschr( full_path + (hasDrive ? 2 : 0), L':') != NULL ||
-        wcsncmp( full_path, L"\\\\?\\", 4) == 0 ||
-        wcsncmp( full_path, L"\\\\.\\", 4) == 0 ||
-        wcspbrk(full_path, L"?*\x1\x2\x3\x4\x5\x6\x7\x8\x9"
-            L"\xA\xB\xC\xD\xE\xF\x10\x11\x12\x13\x14\x15"
-            L"\x16\x17\x18\x19\x1A\x1B\x1C\x1D\x1E\x1F\0") != NULL) {
+    if (wcschr( full_path + (hasDrive ? 2 : 0), W(':')) != NULL ||
```

Comment:
This and other Windows-path specific code in this file should be under #ifndef PLATFORM_UNIX